### PR TITLE
Change page title 

### LINKED ISSUE #24: Change the title of the page
The title of the page could be changed to "Event Calendar".

### PR DESCRIPTION
**Pull request by developer-diganta**
Changed the page title to "Event Calendar" (Closes #24 )

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/EventCalendar/pull/25"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/EddieHubCommunity/EventCalendar)
+
+
```
Comment (NS007-dev) [inline README.md]: @developer-diganta do we need the extra line on line 7?

**Review by NS007-dev — CHANGES_REQUESTED** [review]

**Review by NS007-dev — CHANGES_REQUESTED** [review]
Please can you remove the `package.json` from your commits?

**Comment by developer-diganta** [comment]
@CodingSpecies Thank You for pointing it out. I have removed it.

File: README.md (hunk 1 above)
Comment (developer-diganta) [reply README.md]: No I guess not. Should I remove it?

**Comment by developer-diganta** [comment]
@CodingSpecies Thank You for pointing it out. I have removed it. I deleted it. But I think this would not work?

File: README.md (hunk 1 above)
Comment (NS007-dev) [reply README.md]: Yes please @developer-diganta

**Comment by NS007-dev** [comment]
> @CodingSpecies Thank You for pointing it out. I have removed it. I deleted it. But I think this would not work?

In the LinkFree repo in EddieHub, the package.json is always altered, so we tell them to take it out of the commits. We will see what the maintainers say. Thank you.

**Comment by eddiejaoude** [comment]
Great collaboration 👍 

Deleting it is not a good idea, it is best to put it back to what it was, I think you can use

```
git checkout origin/master package-lock.json
```

**Comment by NS007-dev** [comment]
@eddiejaoude so in this repo do we not need to remove the package.json? Unless it is needed for this pull request?

**Comment by eddiejaoude** [comment]
We need to keep the `package.json` and `package-lock.json` files in the repo, but people should not commit these unless they made changes to the `package.json` file

**Comment by NS007-dev** [comment]
Ok thank you

**Comment by developer-diganta** [comment]
Should I change anything or wait for maintainers to instruct?

**Comment by eddiejaoude** [comment]
@developer-diganta https://github.com/EddieHubCommunity/EventCalendar/pull/25#issuecomment-962280718

**Comment by developer-diganta** [comment]
"error: pathspec 'origin/master' did not match any file(s) known to git " I get this error.. What can I do to resolve it?

**Comment by mohan-murali** [comment]
> "error: pathspec 'origin/master' did not match any file(s) known to git " I get this error.. What can I do to resolve it?

We dont have a master branch. We have a main branch 😄

**Comment by developer-diganta** [comment]
@mohan-murali Thank you so much for pointing it out! I have done it....

**Comment by eddiejaoude** [comment]
Great collaboration everyone.

Thanks for the updates @developer-diganta , we can merge once the `package-lock` file is removed from the PR

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Please only include the relevant changes, see my previous comment

**Comment by developer-diganta** [comment]
Sorry! Got a bit busy with college work.. I wanted to know how to remove the file from the pr?

**Comment by eddiejaoude** [comment]
No problem. 

I added a comment previously https://github.com/EddieHubCommunity/EventCalendar/pull/25#issuecomment-962280718

**Comment by developer-diganta** [comment]
@eddiejaoude  I have run that command. Is there anything else that I should do?

**Review by suitupstan — COMMENTED** [review]
## Merge Conflict

Thank you for your contribution. Your pull request currently has conflicting changes with our `main` branch. In order to review this pull request, we will need those conflicts to be resolved.

We recommend doing this with the `git rebase` command. If you are unsure of how to fix this, let us know and we would be happy to assist you.

**Comment by suitupstan** [comment]
> @eddiejaoude I have run that command. Is there anything else that I should do?

@developer-diganta An easy way would be to just copy-paste the file from this repo, Created PR: https://github.com/developer-diganta/EventCalendar/pull/1

**Comment by suitupstan** [comment]
Hey there @developer-diganta 👋 Just checking in, do you need any help?

File: package-lock.json [hunk 2]
```diff
@@ -36161,12 +35863,6 @@
         "is-typedarray": "^1.0.0"
       }
     },
-    "typescript": {
```
Comment (Cahllagerfeld) [inline package-lock.json:36164]: I think this Lockfile now is fine. It gots cleaned up. Here for some reason is typescript for example

**Review by Cahllagerfeld — COMMENTED** [review]
Small comment

**Review by NS007-dev — COMMENTED** [review]
@developer-diganta any help needed? This pr has not been very active recently. Thank you.

**Comment by developer-diganta** [comment]
Sorry! I had not seen this for quite some time now. I think the changes this PR targeted has already been made. Shall I close this PR?

**Pull request closed without merging**